woocommerce/woocommerce-gutenberg-products-block
# Update the border colors in the Cart and Checkout blocks

## What

Fixes #11354

## Why

In this PR, we're updating the border colors in the Cart and Checkout blocks to match the order confirmation block template defaults and create a cohesive experience.

## Testing Instructions

1. Go to the Cart block page and confirm the border color is similar to #E7E7E7.
2. Open the shipping calculator and confirm the form fields have a border color similar to #1E1E1E. 
3. Go to the Checkout block page and confirm the border color is similar to #E7E7E7
4. Confirm the form fields have a border color similar to #1e1e1e. 
5. Test it with different themes different background colors, and confirm that the borders are visible. 



## Screenshots or screencast


https://github.com/woocommerce/woocommerce-blocks/assets/11503784/85714563-e608-474d-8e7b-92fb4351813e





## WooCommerce Visibility


Required:


## Checklist

Required:


Conditional:


## Changelog


> Update the border colors in the Cart and Checkout blocks.
